docs/templates/ADOC_global.tmpl

```diff
@@ -0,0 +1,30 @@
+[caption=]
+.Environment variables with global scope available in multiple services
+[width="100%",cols="30%,20%,~,~,~",options="header"]
+|===
+| Name
+| Services
+| Type
+| Default Value
+| Description
+
+{{ range . }}
+
+a| `{{ .Name }}`
+
+a| [subs=attributes+]
+{{- range .Services}}
+xref:{s-path}/{{ . }}.adoc[{{ . }}] +
```

Comment:
```suggestion
* xref:{s-path}/{{ . }}.adoc[{{ . }}] +
```
When adding a * it makes a list item out of it. Then readability improves a lot when rendered.